sklearn/utils/tests/test_multiclass.py

```diff
@@ -112,9 +117,13 @@
         np.array([[0, .5], [.5, 0]]),
         np.array([[0, .5], [.5, 0]], dtype=np.float32),
         np.array([[0, .5]]),
+        csr_matrix(np.array([[0, .5], [.5, 0]])),
+        csr_matrix(np.array([[0, .5], [.5, 0]]), dtype=np.float32),
+        csr_matrix(np.array([[0, .5]])),
```

Comment:
I am kind of confused about the cases of `np.array([[0, .5]])` and `csr_matrix(np.array([[0, .5]]))`, these two cases have only one row, but shouldn't it needs to have at least two rows and two columns? 

From what I read from the documentation and the comment, it says ‘continuous-multioutput’: y is a 2d array of floats that are not all integers, and both dimensions are of size > 1.